test/run/ch-fromhost.bats

```diff
@@ -31,6 +31,7 @@ fromhost_ls () {
 
 @test 'ch-fromhost (Debian)' {
     scope standard
+    builder_required
```

Comment:
Can this be covered with `prerequisites_ok`? Because if there is no builder, then the image won't be built. Let's discuss.

Reply:
I'm not sure `prerequisites_ok` works anymore. The only sentinel tarball that is generated is `missing.pq_missing`, other dockerfiles/build are just not included in the generation of build|run_auto.bats anymore.